### PR TITLE
Update arcsight esm

### PR DESCRIPTION
**Pull request by anara123**

### CODE REVIEW

File: Integrations/integration-ArcSightESM.yml [hunk 1]
```diff
@@ -82,13 +82,101 @@ script:
...
+            arrSrc = [arrSrc];
+        }
 
-    var sendAndParse = function(url, args) {
-      return JSON.parse(x2j(sendRequest(url, args)));
+        if (!translationMap) {
+            throw 'translationMap must contain map';
+        }
+
+        /*
```
Comment (shaniacht1) [inline Integrations/integration-ArcSightESM.yml]: Is the comment here on purpose?

File: Integrations/integration-ArcSightESM.yml [hunk 2]
```diff
@@ -311,13 +411,118 @@ script:
...
+    var monitorDataTemplateWithEnv ='<soapenv:Envelope xmlns:soapenv=\'http://schemas.xmlsoap.org/soap/envelope/\'  xmlns:ns2=\'http://ws.v1.service.resource.manager.product.arcsight.com/dataMonitorService/\'  xmlns:ns5=\'http://model.v1.service.resource.manager.product.arcsight.com/xsd\'>'+
+                                        '<soapenv:Header/>'+
+                                        '<soapenv:Body>' +
+                                            getMonitorDataTemplate +
+                                        '</soapenv:Body>' +
+                                    '</soapenv:Envelope>';
+
+    function getDataMonitor(authToken, modCount, monitorID){
+      var soapReq = replaceInTemplates(monitorDataTemplateWithEnv, { authToken:authToken, modificationCount : modCount, monitorID : monitorID});
+      //return soapReq;
```
Comment (shaniacht1) [inline Integrations/integration-ArcSightESM.yml]: comment

Comment (anara123) [inline Integrations/integration-ArcSightESM.yml]: will remove

File: Integrations/integration-ArcSightESM.yml (hunk 1 above)
Comment (anara123) [inline Integrations/integration-ArcSightESM.yml]: yes, explaining how the translate works

**Comment by dorsha** [comment]
@anara123 / @shaniacht1 what about this PR?

**Comment by anara123** [comment]
@dorsha I need to fix it and push.

**Comment by Spivaka** [comment]
@shaniacht1 review again and merge if all ok

**Comment by anara123** [comment]
fixes https://github.com/demisto/content/pull/741

**Comment by dorsha** [comment]
@anara123  can we merge this?

**Comment by anara123** [comment]
@dorsha Yes we can merge this

**Comment by anara123** [comment]
@dorsha please approve so I can merge

**Comment by dorsha** [comment]
Done

**Pull request merged**